cli/mender-artifact/copy_test.go

```diff
@@ -196,6 +196,7 @@ func TestCopy(t *testing.T) {
 				assert.Nil(t, os.Remove("testkey"))
 				// Check that the permission bits have been set correctly!
 				pf, err := NewPartitionFile(imgpath+":/etc/mender/testkey.key", "")
+				defer pf.Close()
```

Comment:
Should we also remove it afterwords?

Reply:
No, the `PartitionFile` does only exist temporarily, so there is no point in removing it. A close will do.